cw/cw305/util/device.py

```diff
@@ -42,9 +42,9 @@ def initialize_fpga(self, bitstream, pll_frequency):
   def initialize_scope(self):
     """Initializes chipwhisperer scope."""
     scope = cw.scope()
-    scope.gain.db = 25
+    scope.gain.db = 30
     # Samples per trace - We oversample by 10x and AES is doing ~12/16 cycles per encryption.
-    scope.adc.samples = 180
+    scope.adc.samples = 200
```

Comment:
Do we really need more samples? I would assume that there is no useful information in the last 20 samples. Everything useful for the attack should happen during the first 120 (AES-128) - 160 (AES-256) samples max.

Reply:
Probably not. I will revert this change.

Reply:
Thanks